### ISSUE TITLE
libwasmer_runtime_c_api.so: No such file or directory

### ISSUE THREAD

**Issue by citterio** [ElrondNetwork/elrond-go-scripts-v2#54]
Installing on a fresh Ubuntu 18.04 as per [https://medium.com/@sever.moldovean/how-install-and-maintain-elrond-nodes-95b1b8f9126e](https://medium.com/@sever.moldovean/how-install-and-maintain-elrond-nodes-95b1b8f9126e) stops with:

> Copying libwasmer to your LIB folder...
>
> cp: cannot stat '/home/stone/go/pkg/mod/github.com/!elrond!network/go-ext-wasm@v                                                                            0.1.0/wasmer/libwasmer_runtime_c_api.so': No such file or directory

Changing `go-ext-wasm@v0.1.0` to `arwen-wasm-vm@v0.3.4` at l. 176 in functions.cfg fixes it for me, but I'm not sending a PR as I'm not sure of the underlying cause. I started seeing this issue on Friday 7/2, both on a Contabo VPS and on a dedicated server somewhere else.

**Comment by citterio** [comment]
Looking at a 1.0.85 install (or possibly 1.0.86, can't remember) from a week ago which autoupdated to 1.0.87, it now has different versions of libwasmer_runtime_c_api.so for arwen and for go-ext-wasm. Maybe the update succeeded because the older version was already there?

```
4598656 Feb  4 14:32 '/home/stone/go/pkg/mod/github.com/!elrond!network/go-ext-wasm@v0.1.0/wasmer/libwasmer_runtime_c_api.so'
5007152 Feb  5 20:25 '/home/stone/go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@v0.3.4/wasmer/libwasmer_runtime_c_api.so'

```

**Comment by citterio** [comment]
Fixed by 98b3f9fd489d5c620b117357339f92bd5ce4a6ea. Thanks!
